docs/cne-install.md

```diff
@@ -0,0 +1,177 @@
+# SAT Upgrade with CNE Installer
+
+## Upgrade the System Admin Toolkit Product Stream
+
+Describes how to upgrade the System Admin Toolkit (SAT) product
+stream by using the Compute Node Environment (CNE) installer (`cne-install`).
+
+The CNE installer can only upgrade the SAT product stream for this release. See [Install the System Admin Toolkit Product Stream](install.md) for installation instructions.
+
+Upgrading SAT with `cne-install` is recommended because the process is both automated and logged to help you save time. The CNE installer can be used to upgrade SAT alone or with other supported products. Refer to the [*HPE Cray EX System Software Getting Started Guide (S-8000)*](https://www.hpe.com/support/ex-S-8000) for detailed information about `cne-install` and its options.
```

Comment:
I noticed that this link works: https://www.hpe.com/support/ex-gsg - I'm guessing the "ex-S-8000" link is planned for the future.

Reply:
Gotcha, thanks for the explanation. Does that link work for you? It still doesn't for me, is it just not up yet or would I need some special permissions?

I just ask because I suspect that the link checker workflow that I added in #35 will have a problem as well. If it becomes a big problem we can add exceptions to the link checker.

Reply:
Right now the link won't work for anyone because it's not set up. I'm in charge of updating that link to point to the Papaya version of the Getting Started Guide once that is available on the HPE Support Center with the Papaya release. 

You're probably right that the lint checker will flag it, but can we still merge PRs anyway? I know that for other repos (because my team has set up links similarly in the past) we had to create an exception. I'm wondering if we can avoid that since the plan was to have the URL always be live after the Papaya release. However, I'm not sure how things will change with unified docs in Alice.

Reply:
Yeah, I think I would agree with not creating an exception, since we would have to undo it later. I think we actually do have the power to merge even if the check is failing.